style.css

```diff
@@ -298,3 +298,26 @@ textarea {
 .contact-me button {
   width: 142px;
 }
+
+input::placeholder {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+}
+
+textarea::placeholder {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
- [ ] Please ensure that the background Image for the contact section always remains similar to the Figma design for different screen sizes, you can achieve this by exporting each asset on its own and with CSS using multiple background images and each image with its own position 👍🏼 

- [ ] Please Ensure that the button is on the left side as shown in the Figma design, Please stick to the design as much as possible as stated on the project requirements 💪🏼 

### Your Design
![image](https://user-images.githubusercontent.com/68030297/228286776-7351d737-1d19-4a8d-9be1-0a6a45c2349d.png)


### Figma Design
![Screenshot from 2023-03-28 16-20-12](https://user-images.githubusercontent.com/68030297/228286469-418c73ff-d744-4c2a-9fc8-ee21bf4fded1.png)